docs/fleet/_index.md

```diff
@@ -96,12 +97,17 @@ Machine [configuration](machines/#configure) and machine [code](/build/program/)
 
 ## The Viam mobile app
 
-{{<gif webm_src="/manage/mobile-app-octagon.webm" mp4_src="/manage/mobile-app-octagon.mp4" alt="GIF of red button being pressed and cannon of confetti bot spraying confetti" class="alignright" max-width="200px">}}
+{{<gif webm_src="/fleet/mobile-app-octagon.webm" mp4_src="/fleet/mobile-app-octagon.mp4" alt="GIF of red button being pressed and cannon of confetti bot spraying confetti" class="alignright" max-width="200px">}}
 
 <br>
 
```

Comment:
In addition to the [Viam app](https://app.viam.com), the fully featured web application where you can access all fleet management tools, there is a Viam mobile app.